pkg/logentry/stages/metrics.go

```diff
@@ -164,6 +164,8 @@ func (m *metricStage) Process(labels model.LabelSet, extracted map[string]interf
 			case *metric.Histograms:
 				m.recordHistogram(name, vec, labels, v)
 			}
+		} else {
+			level.Error(m.logger).Log("msg", "source does not exist", "err", fmt.Sprintf("source: %s, does not exist", *m.cfg[name].Source))
```

Comment:
I think we should make this a debug, seems like it could be very verbose. People usually add debug when they want to understand why it's not working.

Reply:
Changed